osu.Game/Screens/Play/HUD/PerformancePointsCounter.cs

```diff
@@ -56,7 +56,7 @@ private void load(BeatmapDifficultyCache difficultyCache)
                 performanceCalculator = gameplayState.Ruleset.CreatePerformanceCalculator();
                 clonedMods = gameplayState.Mods.Select(m => m.DeepClone()).ToArray();
 
-                scoreInfo = new ScoreInfo(gameplayState.Score.ScoreInfo.BeatmapInfo, gameplayState.Score.ScoreInfo.Ruleset) { Mods = clonedMods };
+                scoreInfo = new ScoreInfo(gameplayState.Score.ScoreInfo.BeatmapInfo, gameplayState.Score.ScoreInfo.Ruleset) { Mods = clonedMods, IsLegacyScore = gameplayState.Score.ScoreInfo.IsLegacyScore };
```

Comment:
@ppy/team-client could I get an approval on this out-of-difficulty change? It sets `IsLegacyScore` so that we can distinguish between legacy scores and lazer CL scores in the PP calculator.